communication/marketing-team/social-guidelines.md

```diff
@@ -26,6 +26,10 @@ Kubernetes Organization Members with access to account are:
 - Contributor/SIG blog posts
 - Events
 - KEPs
+  - Only if the following conditions are met:
+  - Author AND at least half of the assignee/reviewer/approver team (total of 3 people at minimum)
```

Comment:
Maybe we make this more explicit? Something like "Tweet *MUST* be approved by Author AND at least..."

Reply:
As named in the KEP metadata?  Is agreement recorded on the PR itself (that seems fine).

Reply:
thanks for your reviews - made some corrections PTAL